lib/constants.js

```diff
@@ -0,0 +1,13 @@
+'use babel'
+
+export const MachineStatusText = {
+  1: 'Offline',
+  2: 'Online',
+  3: 'Connected'
+}
+
+export const MachineStatus = {
+  Offline: 1,
+  Online: 2,
+  Connected: 3
+}
```

Comment:
I especially made this an object to be able to use it like `MachineStatus.Online`, `MachineStatus.Offline` to make it more readable. 

But i guess i can change `MachineStatusText` to be an array.